moveit_ros/move_group/include/moveit/move_group/move_group_capability.h

```diff
@@ -57,14 +57,16 @@ MOVEIT_CLASS_FORWARD(MoveGroupCapability)
 class MoveGroupCapability
 {
 public:
-  MoveGroupCapability(const std::string& capability_name) : node_handle_("~"), capability_name_(capability_name)
+  MoveGroupCapability(const std::string& capability_name) : capability_name_(capability_name)
   {
   }
-
+  
   virtual ~MoveGroupCapability()
   {
   }
 
+  void setNode(const rclcpp::Node::SharedPtr& node);
```

Comment:
This and `setContext()` are only called once before `initialize()`, right? I would pass node and context as parameters of `initialize()` and remove the setters (similar to how the planning adapters are initialized now).